pilot/pkg/bootstrap/server.go

```diff
@@ -557,6 +565,13 @@ func (s *Server) initMCPConfigController(args *PilotArgs) error {
 			configStores = append(configStores, configController)
 			continue
 		}
+		if url.Scheme == mcpScheme || url.Scheme == mcpsScheme {
```

Comment:
So the configSource.Address does not support mcp or mcps. It supports fs:// or simply IP/host:port. 
If you want to use mcps, then you should specify TLS settings in the config source.

Reply:
If I use mcps, it will require users to specify TLS settings, Should we have a default built-in behaviour to improve user experience?